cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1505,6 +1505,11 @@ public static final class Logging {
     public static final String LOG_APPENDER_EXT_DIR = "app.program.log.appender.extensions.dir";
     public static final String LOG_APPENDER_PROPERTY_PREFIX = "app.program.log.appender.system.properties.";
 
+    // iLog appender configs.
+    public static final String I_LOG_APPENDER_ENABLED = "i.log.appender.provider.enabled";
+    public static final String I_LOG_APPENDER_EXT_DIR = "i.log.appender.provider.extensions.dir";
```

Comment:
Please don't introduce a new prefix. General convention is to use "app.program" for anything happening in the program run and no prefix otherwise. I also not sure why would you want separate extension dirs for the two. You may need to have a separate config similar to `app.program.log.appender.provider`, e.g. `log.appender.provider.system`

Reply:
Done